### PR TITLE
removed volunteer channel

### PR DESCRIPTION
**Pull request by theo-commits**

### CODE REVIEW

File: default-config.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "timezone": "America/Monterrey",
         "username": "Buddy Bot",
         "emojiIcon": ":sunglasses:",
-        "channelId": "volunteers",
+        "channelId": "reclass-admin",
```
Comment (takakonishimura) [inline default-config.json]: probably admin-engies!

File: default-config.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
         "jobs": [
             {
                 "cronTime": "00 00 10 * * 1-2",
```
Comment (takakonishimura) [inline default-config.json]: what time do you want? :D @scottjr632 too

**Review by takakonishimura — COMMENTED** [review]
I thinkklkkk channelId is probably
image

**Review by takakonishimura — APPROVED** [review]
approving for now

File: default-config.json [hunk 3]
```diff
@@ -18,15 +18,15 @@
     {
         "timezone": "America/Monterrey",
         "username": "Buddy Bot",
-        "emojiIcon": ":sunglasses:",
+        "emojiIcon": ":partygopher:",
```
Comment (srctl) [inline default-config.json:21]: I hope this works as an emoji 😅

File: default-config.json [hunk 4]
```diff
@@ -18,15 +18,15 @@
...
         "channelId": "admin-engies",
         "jobs": [
             {
-                "cronTime": "00 00 10 * * 1-2",
-                "text": "@channel Don't forget to clock in! :powerup:"
+                "cronTime": "00 00 07 * * 1-5",
+                "text": "@channel Don't forget to clock in! :partyblobcat:"
```
Comment (srctl) [inline default-config.json:26]: Time LGTM though!

**Comment by takakonishimura** [comment]
@theo-bash ready for merge!

**Pull request merged**